### PR TITLE
Fixed comment marks being removed replying

### PR DESCRIPTION
**Pull request by Devorein**
[ x ] Relocated findChildrenByMark from `prosemirror-utils` to `lib/prosemirror`
[ x ] Added view and threads/votes as a dependency for finding inline action counter

**NOTE**: Please test it out on your end as part of the review

### CODE REVIEW

File: lib/prosemirror/findChildrenByMark.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { MarkType, Node } from '@bangle.dev/pm';
```
Comment (mattcasey) [inline lib/prosemirror/findChildrenByMark.ts:1]: Please leave a comment with reference to the source code when copy/pasting code

**Review by mattcasey — CHANGES_REQUESTED** [review]
We should figure out the reason why simply copying code from the library is a fix. This introduces the same method twice in our codebase, since prosemirror-utils is still a dependency, and we also lose any future improvements from the dependency.  It sounds like there is something else going on, since the functions are idempotent and you didn't have to change anything.

**Comment by Devorein** [comment]
Yes please, that would be really helpful. I'm totally bamboozled as to why the copied version is working and not the original one.

File: lib/prosemirror/findChildrenByMark.ts (hunk 1 above)
Comment (Devorein) [reply lib/prosemirror/findChildrenByMark.ts:1]: Thank you. I completely missed that. Adding that part.

**Comment by mattcasey** [comment]
Closing this, it's been fixed in https://github.com/charmverse/app.charmverse.io/pull/712

**Pull request closed without merging**